exercises/swift-scheduling/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
+
+Your task is to convert delivery date descriptions to _actual_ delivery dates, based on when the meeting started.
+
+There are two types of delivery date descriptions:
+
+1. Fixed: a predefined set of words.
+2. Variable: words that have a variable component, but follow a predefined set of patterns.
+
+## Fixed delivery date descriptions
+
+There are three fixed delivery date descriptions:
+
+- `"NOW"`
+- `"ASAP"` (As Soon As Possible)
+- `"EOW"` (End Of Week)
+
+The following table shows how to translate them:
+
+| Description | Meeting start                 | Delivery date                       |
+| ----------- | ----------------------------- | ----------------------------------- |
+| `"NOW"`     | -                             | Two hours after the meeting started |
+| `"ASAP"`    | Before 12:00                  | Today at 17:00                      |
+| `"ASAP"`    | After 12:00                   | Tomorrow at 12:00                   |
```

Comment:
I think people might complain that it's unclear how to categorize exactly 12:00

Reply:
Fair point! I'll update to a less ambiguous time.

Reply:
I meant that between the two categories "Before 12:00" and "After 12:00", the time "exactly at 12:00" isn't contained in either of them 😁

Reply:
Ah! I also think that 12:00 might be ambiguous between AM and PM. I'll tweak it a bit.

Reply:
I've updated